core.md

```diff
@@ -0,0 +1,111 @@
+# Governance of the NeoForged project
+
+The NeoForged project is split between [github](https://github.com/NeoForged) and [discord](https://discord.gg/UvedJ9m). It is a community of volunteers, maintaining a free and open source ecosystem of modifications to the game Minecraft.
+
+### What this document is
+
+This document is:
+ * A non-exhaustive list of the working principles of the project.
+ * Intended to help direct the project in times of uncertainty.
```

Comment:
Direct doesn't seem like a good fit for the sentence. Maybe steer?

Reply:
It's about project direction, so the verb "to direct" seems the best fit here.